conf.py

```diff
@@ -352,8 +352,14 @@ def xapian_run_example_command(ex):
         return "%s -unsafe -target:exe -out:%s.exe %s -r:XapianSharp.dll\n./%s.exe" \
             % (csc, ex, xapian_code_example_filename(ex), ex)
     elif highlight_language == 'java':
-        javac = get_tool_name('JAVAC', 'javac')
-        java = get_tool_name('JAVA', 'java')
+        java_bindings_dir = os.environ.get("JAVA_BINDINGS_DIR")
+        classpath = ''
+        java_library_path = ''
+        if java_bindings_dir is not None:
+            classpath = ' -classpath ' + java_bindings_dir + 'xapian_jni.jar:./code/java/'
```

Comment:
I'm not sure how to do this. 
Would it be something like this: `os.path.join(java_bindings_dir, 'xapian_jni.jar:./code/java/')`? How do we deal with the `./code/java/` part?